src/miner.cpp

```diff
@@ -149,11 +149,7 @@ BlockAssembler::CreateNewBlock(const CScript &scriptPubKeyIn) {
     }
 
     pblock->nTime = GetAdjustedTime();
-    nMedianTimePast = pindexPrev->GetMedianTimePast();
```

Comment:
We're going to have to have a voice call for me to review this. I don't fully understand what changes semantically for locktime stuff.